app/collections/favorites-button.cjsx

```diff
@@ -89,6 +89,9 @@ module?.exports = React.createClass
         className="favorites-button"
         type="button"
         onClick={@toggleFavorite}>
-        <i className={"fa fa-heart#{if favorited then '' else '-o'}"} />
+        <i className="
+          fa fa-heart#{if @state.favorited then '' else '-o'}
```

Comment:
This isn't set on state anymore just from the PromiseRenderer. Can you also delete it from the getInitialState: -> function?